extensions/amp-base-carousel/0.1/test-e2e/test-mixed-lengths.js

```diff
@@ -76,7 +76,7 @@ describes.endtoend(
         .run('should snap on the center point', async () => {
           const el = await getScrollingElement(controller);
           const slides = await getSlides(controller);
-          const scrollAmount = 1 + (slideOneWidth + slideTwoWidth) / 2;
+          const scrollAmount = 1 + slideOneWidth / 2;
```

Comment:
I'm not sure I understand this change. Could you explain? Shouldn't scrolling by `(1 +  (slideOneWidth + slideTwoWidth) / 2 = 501` snap us in the center point of the second slide (with an `x` of `200px` so that the center of the container aligns with the center of the second slide)? And `1 + slideOneWidth / 2 = 301` should snap us in the center point of the first slide (with an `x` of `100px` so that the center of the container aligns with the center of the first slide)?

Am I misunderstanding the test?

Reply:
> And 1 + slideOneWidth / 2 = 301 should snap us in the center point of the first slide

The first slide is 600px and the second is 400px. The carousel starts off with the first slide's 300th pixel centered. Scrolling to the left by 301px would result in the first pixel of the second slide being the centered pixel, so then the scroll-snap-align property will take effect after the scroll and snap the second slide's 200th px to the carousel's center.

Before the bug fix in #31115, scrolling by 501px was the min pixels required to get to the next slide. So after the fix, I still want to scroll to the left by the min amount to test the functionality of the snapping w/ mixed lengths.

Reply:
Ah yes, that makes sense! I assumed we were starting with the first slide flush to the left, but as this is testing center-snapping, that would not be the case. Thanks for explaining!